crates/bevy_sprite/src/tilemap_chunk/mod.rs

```diff
@@ -60,10 +54,36 @@ pub struct TilemapChunk {
     pub alpha_mode: AlphaMode2d,
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct TileData {
+    pub tileset_index: u16,
+    pub color: Color,
+    pub visible: bool,
+}
+
+impl TileData {
+    pub fn from_index(index: u16) -> Self {
+        Self {
+            tileset_index: index,
+            ..default()
+        }
+    }
+}
+
+impl Default for TileData {
+    fn default() -> Self {
+        Self {
+            tileset_index: 0,
+            color: Color::WHITE,
```

Comment:
Why a default of white rather than black or our standard missing data magenta?

Reply:
I think white is the standard default for tinting, since you're effectively leaving the original pixel color untouched. Maybe it makes sense for it to be optional though?